CHANGELOG.md

```diff
@@ -15,6 +15,7 @@
 - Add CI testing for Python 3.10 and 3.11
 - Add new code formatting tool [isort](https://pycqa.github.io/isort/) to standardise the order and formatting of Python module imports
 - Remove Python 2-3 compatability `from __future__` imports
+- Update fastp to use passing filter reads from filtering results, rather than total reads after filtering ([relevant issue](https://github.com/OpenGene/fastp/issues/367))
```

Comment:
Should be in the updated modules section

Reply:
As it's a fix to a new module that's not yet released I just removed this changelog entry completely.